crates/btc-relay/src/ext.rs

```diff
@@ -15,4 +16,13 @@ pub(crate) mod security {
     ) -> Result<bool, DispatchError> {
         <security::Pallet<T>>::parachain_block_expired(opentime, period)
     }
+
+    pub fn insert_ongoing_fork_error<T: crate::Config>() {
+        <security::Pallet<T>>::set_status(StatusCode::Error);
```

Comment:
Is it possible for the parachain to be in shutdown when this happens? If not, please add a comment somewhere to that effect